zulipterminal/model.py

```diff
@@ -610,6 +610,41 @@ def get_other_subscribers_in_stream(self, stream_id: Optional[int]=None,
                     if stream['name'] == stream_name
                     if sub != self.user_id]
 
+    def get_user_info(self, user_id: int) -> Dict[str, Any]:
+        # TODO: Add custom fields later as an enhancement
+        data = next((i for i in self.initial_data['realm_users']
+                     if i['user_id'] == user_id), Dict[str, Any])
+
+        if not data:
+            # If the user is a cross realm bot (e.g., Welcome Bot)
+            data = next((i for i in self.initial_data['cross_realm_bots']
+                         if i['user_id'] == user_id), Dict[str, Any])
```

Comment:
In `self.user_dict`, some parameters like `date_joined`, `timezone`, roles etc. do not exist.
Re the commit text, thanks for the suggestion. I'll update the commit message :sweat_smile:

Reply:
Re the annotation, I'll update it in the coming update. Thanks for the heads up :)

Re the last point, can we improve the `user_name_from_id` to search in `realm_users` too. I'm not sure how runtime-friendly it would be, however.